code/__HELPERS/spatial_info.dm

```diff
@@ -1,8 +1,3 @@
-/turf
-	///what /mob/oranges_ear instance is already assigned to us as there should only ever be one.
-	///used for guaranteeing there is only one oranges_ear per turf when assigned, speeds up view() iteration
-	var/mob/oranges_ear/assigned_oranges_ear
```

Comment:
I moved this because alt clicking on turf's path to go to parent brought me here instead.